cli/cmd/vuln_host.go

```diff
@@ -601,7 +637,7 @@ func hostVulnCVEsTableForSeverity(cves []api.HostVulnCVE, severity string) [][]s
 		return stringToInt(out[i][7]) > stringToInt(out[j][7])
 	})
 
-	return out
+	return out, total-filtered, total
```

Comment:
This return is confusing. If we are returning the total vulnerabilities minus the filtered ones, what exactly are we returning? The unfiltered or filtered? 

I ask this since you call this function like this:
```
output, filtered, total := hostVulnCVEsTableForSeverity(cves, sev)
```

Reply:
Yeh, I had this thought too, it's not returning the number of filtered items. I'll rethink this.